### DISCUSSION TITLE
Do anyone knows where can i find the variable that contains the value? I want to print it

### DISCUSSION THREAD

**Discussion by jeaugohu4** [jomjol/AI-on-the-edge-device#3429] [Q&A]
Do anyone knows where can i find the variable that contains the value? I want to print it and send it by other way.

**Comment by SybexX** [comment]
You can access it from outside with https://jomjol.github.io/AI-on-the-edge-device-docs/REST-API/

If you want to change the firmware, it's best to take a look at the webhook procedure https://github.com/jomjol/AI-on-the-edge-device/blob/main/code/components/jomjol_webhook/interface_webhook.cpp#L29
